qiskit/providers/backend_compat.py

```diff
@@ -13,188 +13,324 @@
+                    qubits,
                 )
-        target.add_instruction(Measure())
-    # parse global configuration properties
-    if hasattr(configuration, "dt"):
-        target.dt = configuration.dt
-    if hasattr(configuration, "timing_constraints"):
-        target.granularity = configuration.timing_constraints.get("granularity")
-        target.min_length = configuration.timing_constraints.get("min_length")
-        target.pulse_alignment = configuration.timing_constraints.get("pulse_alignment")
-        target.acquire_alignment = configuration.timing_constraints.get("acquire_alignment")
-    # If a pulse defaults exists use that as the source of truth
-    if defaults is not None:
-        inst_map = defaults.instruction_schedule_map
-        for inst in inst_map.instructions:
-            for qarg in inst_map.qubits_with_instruction(inst):
+                continue
+            inst_prop, operational = _decode_instruction_property(gate_spec)
+            if filter_faulty and (set.intersection(faulty_qubits, qubits) or not operational):
+                faulty_ops.add((name, qubits))
                 try:
-                    qargs = tuple(qarg)
+                    del prop_name_map[name][qubits]
+                except KeyError:
+                    pass
                 except TypeError:
-                    qargs = (qarg,)
-                # Do NOT call .get method. This parses Qpbj immediately.
-                # This operation is computationally expensive and should be bypassed.
-                calibration_entry = inst_map._get_calibration_entry(inst, qargs)
-                if inst in target:
-                    if inst == "measure":
-                        for qubit in qargs:
-                            if filter_faulty and qubit in faulty_qubits:
-                                continue
-                            target[inst][(qubit,)].calibration = calibration_entry
-                    elif qargs in target[inst]:
-                        if filter_faulty and any(qubit in faulty_qubits for qubit in qargs):
-                            continue
-                        target[inst][qargs].calibration = calibration_entry
-    combined_global_ops = set()
-    if configuration.basis_gates:
-        combined_global_ops.update(configuration.basis_gates)
-    for op in combined_global_ops:
-        if op not in target:
-            if op in name_mapping:
-                target.add_instruction(name_mapping[op], name=op)
-            else:
-                raise QiskitError(
-                    f"Operation name '{op}' does not have a known mapping. Use "
-                    "custom_name_mapping to map this name to an Operation object"
+                    pass
+                continue
+            if prop_name_map[name] is None:
+                prop_name_map[name] = {}
+            prop_name_map[name][qubits] = inst_prop
+        # Measure instruction property is stored in qubit property in IBM
```

Comment:
```suggestion
        # Measure instruction property is stored in qubit property
```

These are public members and we can assume the measure instruction property is stored in the property object in general.
https://github.com/Qiskit/qiskit/blob/7ef9547447c62dc891a49f58a928b1d3a626083f/qiskit/providers/models/backendproperties.py#L433-L455

Reply:
Applied suggestion :)

Reply:
At Line:206 of backend_compat.py, I am not using BackendProperties.readout_{length, error} because readout_length fails for poughkeepsie.